scripts/update_torch_wheels.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# Must be YYYYMMDD format
+set -e
+DATE=$1
+
+DIST_BUCKET="gs://pytorch-tpu-releases/wheels"
+
+TORCH_WHEEL="torch-1.1.0a0+stable-cp35-cp35m-linux_x86_64.whl"
+TORCH_XLA_WHEEL="torch_xla-0.1+nightly-cp35-cp35m-linux_x86_64.whl"
+if [ ! -z "$DATE" ]; then
+  printf 'Which of the following torch wheels would you link to install?\n'
+  gsutil ls "gs://pytorch-tpu-releases/wheels/torch-${DATE}-*"
```

Comment:
This is a bit dangerous as someone might pick torch and torch_xla at different versions, which is almost certainly recipe for disaster ;)
What is the purpose of this? Selecting stable vs. nightly?

Reply:
Yeah, I added that in case something breaks in the latest nightly wheels so that the customer could be able to go back and select a stable version of a previous date's build. Yep, agreed and could just select the latest wheels for a given date since the both wheels should be published at the same time.

Alternatively, could show the wheel creation date and instruct the user to select wheels with the closest created date, or select that ourselves based on the first selection's creation timestamp? What do you think? Thanks!

Reply:
If we are able to easily select the option names available within a given date, ok, otherwise I'd just wire the "nigthly" vs "stable" names in the script, and have an explicit getopt option for them.

Reply:
Updated. PTAL if this is what we want.